packages/graphiql/src/components/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Going to make a few more accessibility improvements here!

Some excellent prior art from our friends at codesandbox:

https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/components/FileTabs/index.tsx

This implementation in sandpack is a great example of an implementation that appears to be WCAG compliant